UnityProject/Assets/Scripts/Player/PlayerUtils.cs

```diff
@@ -76,6 +76,7 @@ public static void DoReport()
 				}
 			}
 		}
-		SoundManager.PlayNetworked("ClownHonk",Random.Range(0.2f,0.5f),true,true);
+		// JESTER
+		//SoundManager.PlayNetworked("ClownHonk",Random.Range(0.2f,0.5f),true,true);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)